src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -123,6 +131,22 @@ public void initDefaultCommand() {
         setDefaultCommand(moveLift());
     }
 
+    public Command foldArms() {
+        return new InstantCommand("Fold Arms", this, () -> {
+            armsPiston.set(Value.kForward);
+        });
+    }
+
+    public Command deployArms() {
+        return new InstantCommand("Deploy Arms", this, () -> {
+            armsPiston.set(Value.kReverse);
+        });
+    }
+
+    public boolean isSpeedLimitHeight() {
```

Comment:
Instead of doing this, override `void periodic()` and store this value into NetworkTables

Reply:
Done